histomicstk/utils/sample_pixels.py

```diff
@@ -7,7 +7,7 @@
 
 def sample_pixels(slide_path, sample_fraction=None, magnification=None,
                   tissue_seg_mag=1.25, min_coverage=0.1, background=False,
-                  sample_approximate_total=None, tile_grouping=256):
+                  sample_approximate_total=None, tile_grouping=256, flag_color_inversion=False):
```

Comment:
I think the parameter name of `flag_color_inversion` doesn't quite fit the style used elsewhere, and, strictly we are inverting the intensities rather than a color in a single band image, so perhaps a better parameter name would just be `invert_image`.

Reply:
Updated the variable name to `invert_image`